### DISCUSSION TITLE
Unable to get org invoices from api

### DISCUSSION THREAD

**Discussion by fshuman-a** [planetscale/discussion#635] [Bug Reports]
Hello,

I think `https://api.planetscale.com/v1/organizations/<org-name>/invoices` has stopped working? I am using a service token which has `read_invoices` and `read_organization`. The service token was recently created (in the last hour). Wondering if there is a lag?

Error: 404 
```
{
    "code": "not_found",
    "message": "Not Found"
}
```

Thanks!

**Comment by mscoutermarsh** [comment]
@fshuman-a Will look into this. There's no lag when adding a service token permission.

If you could send me which org this is for, I can check the permissions. `mike` `@` our domain.

**Reply by fshuman-a** [reply]
Thanks, I've dropped you an email.

**Comment by fshuman-a** [comment]
User error
